src/Components/CreateVideo/CreateVideo.module.css

```diff
@@ -0,0 +1,18 @@
+.renderMedia {
+    width:50vw;
```

Comment:
here `vw` is ok, because you're describing a container which should be a percentage of the screen